docs/pages/components/Breadcrumb.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Breadcrumb
+description: The Breadcrumb component displays a list of links within a hierarchy and allows navigation back through them.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/Breadcrumb/Breadcrumb.tsx
+---
+
+import { Icon, Breadcrumb, BreadcrumbLink, BreadcrumbItem } from '@deque/cauldron-react'
+
+```js
+import {
+  Breadcrumb,
+  BreadcrumbLink,
+  BreadcrumbItem
+} from '@deque/cauldron-react'
+```
+
+## Examples
+
+### Basic
+
+```jsx example
+<Breadcrumb aria-label="Breadcrumb">
+  <BreadcrumbLink href="/one">One</BreadcrumbLink>
+  <BreadcrumbLink href="/two">Two</BreadcrumbLink>
+  <BreadcrumbLink href="/three">Three</BreadcrumbLink>
+  <BreadcrumbItem>Four</BreadcrumbItem>
+</Breadcrumb>
+```
+
+### Custom Separators
+
+```jsx example
+<Breadcrumb aria-label="Breadcrumb" separator={<Icon type="chevron-double-right" />}>
+  <BreadcrumbLink href="#">One</BreadcrumbLink>
+  <BreadcrumbLink href="#">Two</BreadcrumbLink>
+  <BreadcrumbLink href="#">Three</BreadcrumbLink>
+  <BreadcrumbItem>Four</BreadcrumbItem>
+</Breadcrumb>
+```
+
+## Props
+
+### Breadcrumb
+
+<ComponentProps
+  children={true}
+  refType="HTMLElement"
+  props={[
+    {
+      name: 'separator',
+      type: 'React.Element',
```

Comment:
Looks like the type is wrong, because `React.Element` should be valid, see the example:

```jsx
<Breadcrumb aria-label="Breadcrumb" separator={<Icon type="chevron-double-right" />}>
  <BreadcrumbLink href="#">One</BreadcrumbLink>
  <BreadcrumbLink href="#">Two</BreadcrumbLink>
  <BreadcrumbLink href="#">Three</BreadcrumbLink>
  <BreadcrumbItem>Four</BreadcrumbItem>
</Breadcrumb>
```

Reply:
> Looks like the type is wrong, because React.Element should be valid, see the example:

It would render, yes, but users would run into a TypeScript error:
![image](https://user-images.githubusercontent.com/6416791/227233428-e56008cc-2226-42fa-9e74-6380d9131f3c.png)

Reply:
Yup, unfortunately. It's something that we missed originally.